### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Nathan Parisod

### DISCUSSION THREAD

**Discussion by Nat004499** [heig-vd-dai-course/heig-vd-dai-course#343] [Show and tell]
https://github.com/Nat004499/DAI-Lab01.git
git@github.com:Nat004499/DAI-Lab01.git

**Comment by ludelafo** [comment]
Everything seems fine for me except the `pom.xml`. It seems you have removed the initial properties from the following lines:

```xml
<properties>
    <!-- Omitted for brevity -->
</properties>
```

You must keep the initial values to ensure Maven can encode the files correctly (more on this in the next course) and check for the right Java version.
